% start of file `template.tex'.
%% Copyright 2006-2010 Xavier Danaux (xdanaux@gmail.com).
%% Copyright 2010-2011 Mark Liu (markwayneliu@gmail.com).
%% Copyright 2016-2017 Olga Botvinnik (olga.botvinnik@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.

\documentclass[11pt,letterpaper,serif]{moderncv/moderncv}
\usepackage{verbatim}

% moderncv themes
\moderncvstyle{classic}
\moderncvcolor{blue}

\usepackage{lastpage}

% character encoding
\usepackage[utf8]{inputenc}                   % replace by the encoding you are using

% \usepackage[style=numeric-comp,
% sorting=none,defernumbers=true]{biblatex}%mod.
% \DeclareBibliographyCategory{cited}
% \AtEveryCitekey{\addtocategory{cited}{\thefield{entrykey}}}
% \addbibresource{publications.bib}

% adjust the page margins
\usepackage[scale=0.8]{geometry}
%\setlength{\hintscolumnwidth}{3cm}						% if you want to change the width of the column with the dates
%\AtBeginDocument{\setlength{\maketitlenamewidth}{6cm}}  % only for the classic theme, if you want to change the width of your name placeholder (to leave more space for your address details
%\AtBeginDocument{\recomputelengths}                     % required when changes are made to page layout lengths

% --- Begin use with Multibib ----
\usepackage[resetlabels]{multibib}
\newcites{book,article,poster}{{Books},{Journal Articles},{Conference Posters}}

% Number the Publications
\makeatletter
\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
\makeatother
\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}
% --- END use with Multibib ---

% BibTeX - don't print the year for publications that are "in press"
\newcommand{\dontprintyear}[1]{}


% personal data
\firstname{Olga B.}
\familyname{Botvinnik}
\title{Curriculum Vitae}                               % optional, remove / comment the line if not wanted

% \address{}{}{}    % optional, remove the line if not wanted
% \mobile{(omitted for web)}                    % optional, remove the line if not wanted
\homepage{http://olgabotvinnik.com}                % optional, remove the line if not wanted
\email{olga@olgabotvinnik.com}                      % optional, remove the line if not wanted
\social[linkedin][https://www.linkedin.com/in/olgabotvinnik]{olga-botvinnik}
\social[twitter]{olgabot}                             % optional, remove / comment the line if not wanted
\social[github]{olgabot}                              % optional, remove / comment the line if not wanted
% \date{\today}
\quote{Last Updated: \today}                                 % optional, remove / comment the line if not wanted

%\extrainfo{\url{http://markliu.me}} % optional, remove the line if not wanted

% to show numerical labels in the bibliography; only useful if you make citations in your resume
%\makeatletter
%\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
%\makeatother

% Could never get the page numbers to work so added them by hand with \rfoot
% \nopagenumbers{}                             % uncomment to suppress automatic page numbering for CVs longer than one page
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}

% \input{filename}

\maketitle
\rfoot{\addressfont\itshape\textcolor{gray}{Page \thepage\ of \pageref{LastPage}}}


\cvline{}{Studying the ``dark matter'' of genomes since 2017. PhD in single-cell RNA-sequencing. ``Grand unified theory of cells,'' hashing, sequencing weird creatures, STEM outreach. MIT, Broad Institute, UCSC, UCSD, Chan Zuckerberg Biohub. Open science/data/source.}



\section{Experience}

\cventry{Sep 2024 -- Present}{Seanome}{}{San Francisco, CA}{Founder}{
We build open genomics tools to unlock breakthroughs in ocean diversity.    
}

\cventry{Jan 2022 -- Sep 2024}{Bridge Bio Pharma}{Computational Genomics}{San Francisco, CA}{Senior Computational Biologist}{
Analyzed next-generation sequencing data to prioritize therapeutic targets for genetic diseases with unmet needs.
\begin{itemize}
    \item Analyzed PacBio long-read IsoSeq data of transcriptomes in cell models using Python (\texttt{gffutils, pandas, scikit-learn, Biopython, sourmash} in Jupyter Notebook).
    \item Wrote Nextflow Workflows for long- and short-read sequencing data to support gene therapy programs in capsid evolution and AAV population sequencing.
    \item Deployed Nextflow Workflows on Amazon Web Services Batch.
\end{itemize}
}

\cventry{Jul -- Oct\\2021}{Arcadia Science}{}{San Francisco, CA}{Advisor}{
Provided advice to startup founders on data science and bioinformatics needs.
}

\cventry{2017--2021}{Chan Zuckerberg Biohub}{Data Sciences Platform}{San Francisco, CA}{Bioinformatics Scientist II}{
    \begin{itemize}
    \item Contributed to \href{https://github.com/czbiohub/tabula-muris}{\emph{Tabula Muris}}, \href{https://tabula-muris-senis.ds.czbiohub.org/}{\emph{Tabula Muris Senis}}, 
        \href{https://tabula-sapiens.sf.czbiohub.org/}{\emph{Tabula Sapiens}}, and \href{http://tabula-microcebus.ds.czbiohub.org/}{\emph{Tabula Microcebus}} 
        single-cell atlases.
    \item Contributed Python and Rust code to ``sourmash'' software for genomic and metagenomic $k$-mer analyses 
        (\href{https://github.com/dib-lab/sourmash}{github.com/dib-lab/sourmash})
    \item Led cross-species analyses projects to study ``dark matter'' of transcriptomes using subsampled $k$-mers from translated RNA-seq reads 
        in a reduced amino acid alphabet, to find putative homologous genes across millions of years of evolution 
            (\href{https://czbiohub.github.io/de-novo-orthology-paper/}{open draft manuscript})
    \item Pioneered weekly weekly internal ``Cupcakes \& Coding'' sessions, alternating between facilitating tutorials taught by junior programmers 
        to share coding knowledge, and pair programming sessions (\href{https://github.com/czbiohub/cupcakes}{github.com/czbiohub/cupcakes}) 
    \item Created \href{https://twitch.tv/bioinformaticsbeyonce}{``BioinformaticsBeyonce''} Twitch channel showcasing open source bioinformatics tools and research 
    \end{itemize}
% Projects:
% \begin{itemize}
% \item Analyzed large single-cell transcriptome of mouse organs in collaboration with 50+ domain experts on \emph{Tabula Muris} resource (\href{https://github.com/czbiohub/tabula-muris}{github.com/czbiohub/tabula-muris}), published in \emph{Nature}
% \item Collaborated with a multidisciplinary team on \emph{Tabula Microcebus} the first single-cell atlas of a nonhuman primate, Mouse Lemur (\href{http://tabula-microcebus.ds.czbiohub.org/}{tabula-microcebus.ds.czbiohub.org})
% % \item Wrote utility function in Rust to count $k$-mers in sequence data (\href{https://github.com/czbiohub/extract_kmers}{github.com/czbiohub/extract\_kmers})
% \item Contributed Python and Rust code to ``sourmash'' software for genomic and metagenomic $k$-mer analyses (\href{https://github.com/dib-lab/sourmash}{github.com/dib-lab/sourmash})
% \item Developed ``orpheum'' (\href{https://github.com/olgabot/orpheum}{github.com/olgabot/orpheum}) tool to extract likely protein-coding reading frames from RNA-seq data.
% \item Led cross-species analyses projects to study ``dark matter'' of transcriptomes using subsampled $k$-mers from translated RNA-seq reads in a reduced amino acid alphabet, to find putative homologous genes across millions of years of evolution (\href{https://czbiohub.github.io/de-novo-orthology-paper/}{open draft manuscript})
% \item Pioneered weekly ``BioinformaticsBeyonce'' Twitch channel showcasing open source bioinformatics tools and research (\href{https://twitch.tv/bioinformaticsbeyonce}{twitch.tv/bioinformaticsbeyonce})
% \item Spearheaded weekly internal ``Cupcakes \& Coding'' sessions, alternating between facilitating tutorials taught by junior programmers to share coding knowledge, and pair programming sessions (\href{https://github.com/czbiohub/cupcakes}{github.com/czbiohub/cupcakes})
% \end{itemize}
}

\cventry{2010--2011}{Jill Mesirov Laboratory}{Broad Institute of Harvard and MIT}{Cambridge, MA}{Bioinformatics Research Assistant}{Created REVEALER algorithm to unveil candidate oncogenic activators, published in \emph{Nature Biotechnology}. Contributed to papers published in \emph{Science Signaling} and \emph{Journal of Hematology \& Oncology}.}


\section{Education}
\cventry{2017}{Ph.D., Bioinformatics and Systems Biology}{University of California, San Diego}{La Jolla, CA}{}{\href{https://github.com/olgabot/dissertation}{Dissertation}: Computational analysis of single-cell alternative splicing} % arguments 3 to 6 can be left empty
\cventry{2012}{M.S., Bioinformatics and Biomolecular Engineering}{University of California, Santa Cruz}{Santa Cruz, CA}{}{}
% \cvline{advisor:}{\small Professor Nader Pourmand}
\cventry{2010}{S.B., Biological Engineering}{Massachusetts Institute of Technology}{Cambridge, MA}{}{}
\cventry{2010}{S.B., Mathematics}{Massachusetts Institute of Technology}{Cambridge, MA}{}{}
% \cvline{gpa:}{\small 3.76/4.0}
% \cvline{honors:}{\small Cum Laude}

\section{Research Training}

\cventry{2013--2017}{Gene Yeo Laboratory}{University of California, San Diego}{La Jolla, CA}{}{Led machine-learning analyses of single-cell motor neuron differentiation mRNA-seq data, in collaboration with wet-lab researchers. Independently developed several software packages written in Python for alternative splicing analyses}

\cventry{2012--2013}{Research Rotations}{University of California, San Diego}{La Jolla, CA}{}{Worked in Profs. Trey Ideker, Gene Yeo, and Pavel Pevznerâ€™s laboratories}

\cventry{2012}{Nader Pourmand Laboratory}{University of California, Santa Cruz}{Santa Cruz, CA}{}{Analyzed single-cell response of breast cancer drug resistance to paclitaxel}

\cventry{2010}{Sebastian Seung Laboratory}{MIT Department of Brain and Cognitive Sciences}{Cambridge, MA}{}{Computed directionality of neurons in electron microscopy of rabbit retina}

\cventry{2009}{David Gifford Laboratory}{MIT Computer Science and Artificial Intelligence Laboratory}{Cambridge, MA}{}{Tested whether information flow can predict gene lethality in genomic networks}

\cventry{2008}{Sean Eddy Laboratory}{Howard Hughes Medical Institute Janelia Farm Research Campus}{Ashburn, VA}{}{Used Hidden Markov Models to improve protein homology search with robust null models}

\cventry{2007}{Martha Bulyk Laboratory}{Brigham and Women's Hospital, Division of Genetics}{Boston, MA}{}{Analyzed DNA binding specificities of mouse homeodomain transcription factors}


\section{Awards}
\subsection{Fellowships}
\cvline{2017}{University of Washington eScience Institute - Moore/Sloan Data Science and Washington Research Foundation Innovation in Data Science Postdoctoral Fellowship (declined)}
\cvline{2014}{NumFocus John Hunter Technical Fellowship for Open Source Science}
\cvline{2013--2016}{National Defense Science and Engineering Graduate Fellowship}

\subsection{Honors}
\cvline{2016}{100 Awesome Women In The Open-Source Community You Should Know, sourced.com}
\cvline{2013}{Fannie and John Hertz Foundation Fellowship Finalist}
\cvline{2012}{National Science Foundation Graduate Research Fellowship: Honorable Mention}
\cvline{2012}{University of California Regents Scholarship}
\cvline{2009}{Bernard M. Gordon-MIT Engineering Leadership Program}
\cvline{2008}{Howard Hughes Medical Institute Janelia Farm Research Summer Scholar}


% --- START For use with BibLatex ---
% \begin{refsection}[publications]
% \nocite{*}
% \printbibliography[resetnumbers=true,sorting=ynt,%mod
% title={Entire publication list sorted by year}]
% \end{refsection}
% ---- END for use with biblatex ----

% --- START For use with BibTeX, not BibLaTeX .... ---
% \nocite{*}
% \bibliographystyle{habbrvyr}
% % \biblographystyle{plain}
% \bibliography{publications}                    % 'publications' is the name of a BibTeX file

% % \nocite{*}
% % \bibliographystyle{habbrvyr}
% % \bibliography{books}
% ---- END for use with bibtex ----


% --- START for use with multibib package ---
% Publications from a BibTeX file using the multibib package
\section{Publications}

\nocitearticle{*}
\bibliographystylearticle{habbrvyrolgabold}
\bibliographyarticle{articles}                   % 'articles' is the name of a BibTeX file

\nocitebook{*}
\bibliographystylebook{habbrvyrolgabold}
\bibliographybook{books}                   % 'books' is the name of a BibTeX file

\nociteposter{*}
\bibliographystyleposter{habbrvyrolgabold}
\bibliographyposter{posters}                   % 'posters' is the name of a BibTeX file
% --- END for use with multibib package ----

\section{Talks}

\cventry{2020}{Biology of Genomes}{Cold Spring Harbor Laboratory}{Long Island, New York}{Functional prediction of transcriptomic ``dark matter'' across species}{Slides: {\footnotesize \href{https://speakerdeck.com/olgabot/functional-prediction-of-transcriptomic-dark-matter-across-species}{speakerdeck.com/olgabot/functional-prediction-of-transcriptomic-dark-matter-across-species}}}

\cventry{2019}{The Identity and Evolution of Cell Types}{European Molecular Biology Laboratory}{Heidelberg, Germany}{Reference-free comparative transcriptomics}{Slides: {\footnotesize \href{https://speakerdeck.com/olgabot/reference-free-comparative-transcriptomics}{speakerdeck.com/olgabot/reference-free-comparative-transcriptomics}}}
\cventry{2019}{Data Intensive Biology Summer Institute (Invited speaker)}{University of California, Davis}{Davis, CA}{Single-cell RNA-seq: To Infinity and Beyond!}{Slides: {\footnotesize \href{https://osf.io/gdzuy/}{osf.io/gdzuy/}}, Video: {\footnotesize \href{https://youtu.be/hAqa8DztxSU}{youtu.be/hAqa8DztxSU}}}
\cventry{2018}{Biological Data Science}{Cold Spring Harbor Laboratories}{Cold Spring Harbor, NY}{Fast approximate cell type identification via MinHash sketches of k-mers in single cell RNA-seq}{Slides: {\footnotesize \href{https://docs.google.com/presentation/d/1JXt4p5eJDjUEV1fD6V9CejyyCs3N8un2tVZ0ihadDW0/edit?usp=sharing}{Google slides}}}
\cventry{2018}{Current Progress in Biotechnology Seminar Series (Invited speaker)}{University of California, Davis}{Davis, CA}{If you liked it, you should have put a Seq on it: Job-seq and lessons learned}{Slides: {\footnotesize \href{https://www.slideshare.net/olgabotvinnik/if-you-liked-it-you-should-have-put-a-seq-on-it}{slideshare.net/olgabotvinnik/if-you-liked-it-you-should-have-put-a-seq-on-it}}}
\cventry{2017}{Open Data Science Conference}{San Francisco Hyatt Regency}{San Francisco, CA}{Co-evolution of algorithms and data in biology}{Slides:
{\footnotesize \href{https://speakerdeck.com/olgabot/co-evolution-of-algorithms-and-data-in-biology}{\texttt{speakerdeck.com/olgabot/co-evolution-of-algorithms-and-data-in-biology}}}}
\cventry{2016}{Festival of Genomics California}{San Diego Convention Center}{San Diego, CA}{}{}
\cventry{2016}{Fluidigm User Group Meeting}{City of Hope Hospital}{Los Angeles, CA}{}{}
\cventry{2016}{Bioiformatics and Systems Biology Bootcamp}{University of California, San Diego}{La Jolla, CA}{Dr. You or How I Learned to Stop Worrying and Love the Ph.D.}{Slides: {\footnotesize \href{http://www.slideshare.net/olgabotvinnik/dr-you-or-how-i-learned-to-stop-worry-and-love-the-phd}{\texttt{slideshare.net/olgabotvinnik/dr-you-or-how-i-learned-to-stop-worry-and-love-the-phd}}}}
\cventry{2016}{Bioinformatics and Systems Biology Ph.D. Program Recruitment}{University of California, San Diego}{La Jolla, CA}{}{}
\cventry{2015}{Bioiformatics and Systems Biology Bootcamp}{University of California, San Diego}{La Jolla, CA}{Dr. You or How I Learned to Stop Worrying and Love the Ph.D.}{Slides: {\footnotesize \href{http://www.slideshare.net/olgabotvinnik/dr-you-or-how-i-learned-to-stop-worry-and-love-the-phd}{\texttt{slideshare.net/olgabotvinnik/dr-you-or-how-i-learned-to-stop-worry-and-love-the-phd}}}}
\cventry{2015}{San Diego Bioinformatics User Group}{University of California, San Diego}{La Jolla, CA}{Open-source software for single-cell and other large-scale transcriptomic datasets}{Slides: \url{http://nbviewer.jupyter.org/format/slides/gist/olgabot/2ee1087d74df46c842df/} (same as below)}
\cventry{2015}{CodeNeuro}{New Museum}{New York, NY}{Flotilla: Data-driven conversations in biology}{Slides: \url{http://nbviewer.jupyter.org/format/slides/gist/olgabot/2ee1087d74df46c842df/}}
\cventry{2015}{AmpNeuro}{Amplifying Neuroscience Symposium}{La Jolla, CA}{Open-source software for single-cell and other large-scale transcriptomic datasets}{Slides: \url{http://nbviewer.jupyter.org/format/slides/gist/olgabot/ba6970fbfa2babd79f55/}}
\cventry{2015}{Bioinformatics Exchange}{University of California, San Diego}{La Jolla, CA}{}{}
\cventry{2015}{Bioinformatics and Systems Biology Ph.D. Program Recruitment}{University of California, San Diego}{La Jolla, CA}{}{}
\cventry{2014}{RNA Club}{University of California, San Diego}{La Jolla, CA}{}{}
\cventry{2014}{Bioinformatics EXPO}{University of California, San Diego}{La Jolla, CA}{}{Best Talk, 2nd place}
\cventry{2014}{PyData}{401 Park Ave. South}{New York, NY}{}{Presentation: \url{https://www.youtube.com/watch?v=IQksDvFl2_8}. Slides: \url{https://github.com/olgabot/pydata2014biodata}}

\section{Teaching, Outreach, and Leadership}

% \cventry{year--year}{Job title}{Employer}{City}{}{Description}
\cventry{2018--}{Core member}{nf-core}{\url{nf-co.re}}{}{Contributed to a community effort to collect a curated set of analysis pipelines built using Nextflow.}

\cventry{2017--2019}{Dancer}{SoulForce Dance Company}{San Francisco, CA}{}{Member of 11-piece hip-hop dance company. Performed in twice annual shows and assisted in beginner dance classes.}


\cventry{2015--2017}{Principal Cellist}{UCSD Chamber Orchestra}{La Jolla, CA}{}{First cellist out of six. Led cello section during rehearsals.}


\cventry{2016-2017}{Teaching Assistant}{Cold Spring Harbor Laboratories}{Cold Spring Harbor, NY}{}{
Developed and led bioinformatics coursework of \emph{Single Cell Analysis Course} including alignment, machine learning, Python, and basic command line tools to an audience largely with little to no programming experience. Course materials available at \url{http://github.com/YeoLab/single-cell-bioinformatics}
}

\cventry{2016}{Guest Instructor}{Quantitative Methods in Genetics and Genomics}{La Jolla, CA}{
}{Taught three weeks of \texttt{git}, RNA-seq and analysis methods to graduate-level UCSD course of 30 students, mostly with limited programming experience. Course materials available at \url{http://github.com/biom262/biom262-2016}}

\cventry{2015--2016}{Speaker and Co-Organizer}{CodeNeuro}{New York, NY and San Francisco, CA}{}{
Presented \texttt{flotilla} software, taught ``coding for neuroscientists'' tutorial (\url{http://github.com/codeneuro/gitgoing}), and advanced data analysis tutorial}

\cventry{2015--2016}{President and Co-Founder}{Graduate Bioinformatics Council}{La Jolla, CA}{}{
Founded graduate student council organization for UCSD Bioinformatics and Systems Biology Program. Advocated for student voices, organized ``town hall'' meetings, social hours, fellowship peer review, and led a team of eight vice presidents and representatives.
}

\cventry{2015}{Guest Instructor}{Quantitative Methods in Genetics and Genomics}{La Jolla, CA}{
}{Taught ``data cleaning'' and plotting course using Python to graduate-level UCSD course of 10 students, mostly with limited programming experience.}


\cventry{2013--2016}{Volunteer}{San Diego Science and Engineering Festival}{San Diego, CA}{}{Developed and demonstrated bioinformatics modules to all ages at UCSD Bioinformatics booth.}

\cventry{2013--2014}{Instructor}{Bioinformatics Algorithms}{Coursera.org}{}{Developed interactive curriculum for online Bioinformatics Algorithms Coursera class and textbook. Advisors: Pavel Pevzner and Phillip Compeau}

\cventry{2011--2012}{Mentor}{We Teach Science}{San Jose, CA}{}{Weekly algebra tutoring to an 8th grader}

\cventry{2011--2012}{Guest Instructor}{Pacific Collegiate School}{Santa Cruz, CA}{}{Created bioinformatics modules to engage students in tying genotype to phenotype for high school AP Biology}

\cventry{2012}{Co-Chair}{Intelligent Systems for Molecular Biology Student Council Symposium}{}{Long Beach, CA}{}

\cventry{2012}{Instructor}{Minority Access to Research Careers}{Santa Cruz, CA}{}{Taught inquiry-based stem cell bioinformatics curriculum to undergraduate researchers}
\cventry{2011}{Volunteer}{Science Club for Girls}{Cambridge, MA}{}{Co-led after-school biology science club for a class of 16 2nd graders}

\cventry{2009--2011}{Choreographer}{MIT DanceTroupe}{Cambridge, MA}{}{Taught beginner to intermediate hip-hop choreography to fellow students}
\cventry{2008--2010}{Publicity Chair}{MIT DanceTroupe}{Cambridge, MA}{}{Designed posters and T-shirts to publicize and promote DanceTroupe concert attendance}
\cventry{2008}{Social Chair}{Baker House}{Cambridge, MA}{}{Organized social events for students, including a popular ``Dormal'' event with catered dinner and jazz music performances}


\section{Software}
\cvline{}{All software is written in Python and open source, licensed under the 3-clause BSD license, except where noted.}

\cvline{\texttt{anchor}}{Categorizes alternative splicing data into ``modes''---bimodal, unimodal, or uniform. \url{http://github.com/YeoLab/anchor}}

\cvline{\texttt{bonvoyage}}{Transforms 1d splicing profiles into 2d space to maximize interpretability of change in signal. \url{http://github.com/YeoLab/bonvoyage}}

\cvline{\texttt{dobby}}{Dobby is a free and open source package for converting and managing plate reader fluorescence outputs, cDNA concentration files, ECHO pick lists, and creating sample sheets for Illumina sequencing.
 \url{http://github.com/czbiohub/dobby}}

\cvline{\texttt{flotilla}}{All-in-one package to perform machine learning analyses on large-scale molecular profiling datasets such as gene expression and alternative splicing. \url{http://github.com/YeoLab/flotilla} (72 stars on GitHub)}

\cvline{\texttt{hermione}}{Compare multiple distributions with horizon plots (also known as ridge plots) \url{http://github.com/czbiohub/hermione} (6 stars on GitHub)}

\cvline{\texttt{kvector}}{Counts $k$-mers in DNA or RNA as $k$-mer vectors, transforms position weight matrices (PWMs) to $k$-mer vectors. \url{http://github.com/olgabot/kvector} (5 stars on GitHub)}


\cvline{\texttt{nf-core}}{A collection of high quality Nextflow pipelines. Contributed to \texttt{rnaseq}, \texttt{scrnaseq} and wrote \texttt{kmermaid pipeline} \url{http://github.com/nf-core}}

\cvline{\texttt{outrigger}}{Fast \emph{de novo} alternative exon detection and quantification. \url{http://github.com/YeoLab/outrigger} (9 stars on GitHub)}

\cvline{\texttt{qtools}}{Submit jobs to the supercomputer cluster from within Python. \url{http://github.com/YeoLab/qtools} (11 stars on GitHub)}

\cvline{\texttt{poshsplice}}{Annotates alternative splicing events with biological features such as translated protein product. \url{http://github.com/olgabot/poshsplice} (2 stars on GitHub)}

\cvline{\small\texttt{prettyplotlib}}{Painlessly create beautiful \texttt{matplotlib} plots.
% prettyplotlib url is too long for the page width and looks weird so using the small font here
\newline\url{http://github.com/olgabot/prettyplotlib} (1,159 stars on GitHub)}

\cvline{\texttt{pyhomer}}{Utility functions to work with output from the HOMER motif finding program. \url{https://github.com/olgabot/pyhomer}}

\cvline{\texttt{seaborn}}{Statistical visualization library. Contributor, wrote clustered heatmap classes and function. \url{http://github.com/mwaskom/seaborn} (3,602 stars on GitHub)}

\cvline{\texttt{sourmash}}{Compute and compare MinHash signatures for DNA data sets. Contributor, wrote reduced amino acid alphabet and other protein fixes. \url{https://github.com/dib-lab/sourmash} (199 stars on GitHub)}

\cvline{\texttt{wasabiplot}*}{Plot coverage and junction reads for any bam file and any region. *Derivative of SashimiPlot, and thus under the GNU General Public License (GPL). \url{http://github.com/olgabot/wasabiplot}}


\section{Mentees}
% \cventry{2012}{Nader Pourmand Laboratory}{University of California, Santa Cruz}{Santa Cruz, CA}{}{Analyzed single-cell response of breast cancer drug resistance to paclitaxel}
\cventry{2019}{Saba Nafees}{Chan Zuckerberg Biohub}{San Francisco, CA}{PhD Intern}{}
\cventry{2018}{Gerry Meixong}{Chan Zuckerberg Biohub}{San Francisco, CA}{Undergraduate Intern}{}
\cventry{2014--2017}{Jessica Lettes}{University of California, San Diego}{La Jolla, CA}{Undergraduate Student}{}
\cventry{2013 Summer}{Natalia La Spada}{University of California, San Diego}{La Jolla, CA}{High School Student}{}
% \cventry{2014--Present}{Jessica Lettes}

\end{document}
